stdlib/public/Concurrency/TaskGroup.swift

```diff
@@ -290,6 +262,74 @@ public func _unsafeInheritExecutor_withThrowingTaskGroup<ChildTaskResult, GroupR
 /// and mutation operations can't be performed
 /// from a concurrent execution context like a child task.
 ///
+/// Structured Concurrency
+/// ======================
+///
+/// A task group is the primary way to create structured concurrency tasks in Swift.
+/// Another way of creating structured tasks are `async let` declarations.
+///
+/// Structured concurrency tasks are often called "child tasks" because of their relationship with their parent task.
+/// A child task will inherit the parent's priority, task-local values, and will be structured in the sense that its
+/// lifetime will never exceed the lifetime of the parent task.
+///
+/// A task group will *always* wait for all child tasks to complete before it is destroyed.
+/// Specifically any `with...TaskGroup` APIs, will not return until all the child tasks
+/// created in the group's scope have completed running.
+///
+/// Structured concurrency is a way to organize your program, and tasks, in such a way that
+/// tasks do not outlive the scope in which they are created. The guarantee that within
+/// a structured task hierarchy, which can span many levels of child tasks,
+/// no child-task will remain running longer than its parent task, simplifies reasoning about resource usage,
+/// and is a powerful mechanism that you can use to write well-behaved concurrent programs.
+///
+/// Structured Concurrency APIs (including task groups and `async let`), will *always* await the
+/// completion of tasks contained within their scope before returning. Specifically, this means that
+/// even if one were to await a single task result and return it from a `withTaskGroup` function body,
+/// the group will automatically await all the remaining tasks before returning:
+///
+///     func takeFirst(actions: [@Sendable () -> Int]) async -> Int? {
+///         await withTaskGroup { group in
+///             for action in actions {
+///                 group.addTask { action() }
+///             }
+///
+///             return await group.next() // return the first action to complete
+///         } // the group will ALWAYS await the completion of all the actions (!)
+///     }
+///
+/// In the above example, even though we return the first collected integer from all actions added to the task group,
```

Comment:
Avoiding "we" per style guide:

```suggestion
/// In the above example, even though the code returns the first collected integer from all actions added to the task group,
```

Reply:
Marking this unresolved because I still see "even though we return" above — was this not changed?

Reply:
Whoops, I messed up indeed -- reapplying your suggestions, thank you